docs/azure_jumpstart_ag/manufacturing/contoso_motors/ai_inferencing/_index.md

```diff
@@ -0,0 +1,68 @@
+---
+type: docs
+weight: 9
+title: AI Inference
+linkTitle: AI
+---
+
+# Jumpstart Agora - Contoso Motors Web UI & AI Inference
+
+The Contoso Motor AI inference flow was designed with a Kubernetes-based architecutre in mind, providing a flexible and scalable design that can be easily adapted and extended by our customers. This document covers how Contoso Motors implements different AI models depending on the different scenarios. 
+
+## Architecture
+
+![AI inference flow](./img/ai_flow.png)
+
+From the image above, the Edge AI inference flow can be divided into two configuration steps and three inference steps. 
+
+1. **Model Downloader:** this is a *Kubernetes Job* that's in charge of downloading the binary files of the AI models and the corresponding configuration files. Depending on the model type, various formats can be used, such as .onnx, .hd5, .bin, and others. In general, the binary files contain the model weights and architecture, while the configuration files specify the model properties, such as input/output tensor shapes, number of layers, and more. 
+
+    In this scenario, the models and configurations needed for the OVMS deployment are hosted within the Arc Jumpstart **jsfiles** storage account. During deployment, the **Model Downloader** job pulls these models and configurations by running the **ovms_config.sh** script, which downloads all necessary model files and stores them in the  **ovms-pvc** persistent volume claim for the OVMS pods to access and serve. All models need to be placed and mounted in a particular directory structure and according to the rules described in [OpenVINO - Model Serving](https://docs.OpenVINO.ai/2022.3/ovms_docs_models_repository.html). 
```

Comment:
```suggestion
    The models and configurations needed for the OVMS deployment are hosted within the Arc Jumpstart **jsfiles** storage account. During deployment, the **Model Downloader** job pulls these models and configurations by running the **ovms_config.sh** script, which downloads all necessary model files and stores them in the  **ovms-pvc** persistent volume claim for the OVMS pods to access and serve. All models need to be placed and mounted in a particular directory structure and according to the rules described in [OpenVINO - Model Serving](https://docs.OpenVINO.ai/2022.3/ovms_docs_models_repository.html). 
```

Reply:
Have we already introduced OVMS somewhere previously? We need to explain what OVMS is and link out to the official docs

Reply:
Added as part of the intro